collector_k8s/scrape_configs_statefulset.yaml

```diff
@@ -0,0 +1,401 @@
+- job_name: stack-kubelet/0
```

Comment:
Should we drop this `stack-kubelet` job, since this would kinda be specific to the customer env?

Reply:
in this case i added it to help customers running the statefulset get going. but maybe i should remove that and just keep in the otel related ones.